policyengine_us/parameters/gov/states/dc/tax/income/credits/eitc/without_children/phase_in/rate.yaml

```diff
@@ -0,0 +1,15 @@
+description: DC phases in EITC amount at this rate applied to earned income.
+metadata:
+  unit: /1
+  label: DC EITC phase-in rate for filers without qualifying children
+  period: year
+  reference:
+    - title: Code of the District of Columbia § 47–1806.04 (f)(1)(C)(i)
```

Comment:
I don't see 7.65% in this section. If it refers to the IRC, please apply the relevant parameter from the `irs` folder instead.

Reply:
Done.